source/module_elecstate/elecstate_print.cpp

```diff
@@ -173,16 +174,18 @@ void ElecState::print_etot(const bool converged,
 
     if (printe > 0 && ((iter + 1) % printe == 0 || converged || iter == GlobalV::SCF_NMAX))
     {
+        int n_order = std::max(0, Occupy::gaussian_type);
         GlobalV::ofs_running << "\n " << std::setw(12) << "Energy" << std::setw(30) << "Rydberg" << std::setw(30)
                              << "eV" << std::endl;
         this->print_format("E_KohnSham", this->f_en.etot);
+        this->print_format("E_KS(sig->0)", this->f_en.etot - this->f_en.demet/(2+n_order));
         this->print_format("E_Harris", this->f_en.etot_harris);
         this->print_format("E_band", this->f_en.eband);
         this->print_format("E_one_elec", this->f_en.eband + this->f_en.deband);
         this->print_format("E_Hartree", this->f_en.hartree_energy);
         this->print_format("E_xc", this->f_en.etxc - this->f_en.etxcc);
         this->print_format("E_Ewald", this->f_en.ewald_energy);
-        this->print_format("E_entropy (-TS)", this->f_en.demet); // mohan add 2011-12-02
+        this->print_format("E_entro(-TS)", this->f_en.demet); // mohan add 2011-12-02
```

Comment:
E_entro is more unclear than E_entropy. Is there any special reason for this change of naming?